packages/terra-badge/src/Badge.module.scss

```diff
@@ -3,9 +3,10 @@
 
 :local {
   .badge {
+    border: var(--terra-badge-border);
     border-radius: var(--terra-badge-border-radius, 0.25em);
     display: var(--terra-badge-display, inline-block);
-    font-size: var(--terra-badge-font-size-small, 0.8rem);
+    font-size: var(--terra-badge-default-font-size, 0.8rem);
```

Comment:
nit: I'd drop the `default` part to be consistent with the names of the other variables under the .badge class.

Reply:
https://github.com/cerner/terra-core/pull/2133/commits/a72fdf778b090b7cf5ac106e1f1ecd0bd1456945#diff-efdedb295ee6eff1ea893adcb2918e84R9